sunpy/lightcurve/sources/noaa.py

```diff
@@ -44,30 +44,31 @@ class NOAAIndicesLightCurve(LightCurve):
     * `Indices Descriptions <ftp://ftp.swpc.noaa.gov/pub/weekly/README3>`_
     * `NOAA plots of Solar Cycle Progression <http://www.swpc.noaa.gov/products/solar-cycle-progression>`_
     * `NOAA Product List <http://www.swpc.noaa.gov/products-and-data>`_
+
+
     """
+    default_url_noaa_indices = "ftp://ftp.swpc.noaa.gov/pub/weekly/RecentIndices.txt"
 
     def peek(self, type='sunspot SWO', **plot_args):
         """Plots NOAA Indices as a function of time. An example is shown below.
 
-        .. plot::
-
-            from sunpy import lightcurve as lc
-            noaa = lc.NOAAIndicesLightCurve.create()
-            noaa.peek()
+        >>> from sunpy import lightcurve as lc
+        >>> noaa = lc.NOAAIndicesLightCurve.create()
+        >>> noaa.peek()
 
         Parameters
         ----------
         type : str
-            The type of plot required.
+        The type of plot required.
```

Comment:
hmm trying ... it was giving me some sphinx errors back then ... let me see again